x/ccv/provider/keeper/key_assignment_test.go

```diff
@@ -47,7 +47,9 @@ func TestGetAllValidatorConsumerPubKey(t *testing.T) {
 	pk, ctx, ctrl, _ := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
 	defer ctrl.Finish()
 
-	rand.Seed(time.Now().Unix())
+	seed := time.Now().UnixNano()
```

Comment:
By upgrading go to 1.20 for golangci-lint, the tool now enforces we don't use the depreciated way of rand seeding